libs/components/forms/src/lib/modules/radio/radio-group.component.spec.ts

```diff
@@ -721,6 +721,22 @@ describe('Radio group component (reactive)', function () {
         .textContent,
     ).toEqual('This option is incorrect.');
   }));
+
+  it('should render help inline only if label text and help popover content is provided', async () => {
+    componentInstance.labelText = 'Label Text';
+    fixture.detectChanges();
+
+    expect(
+      fixture.nativeElement.querySelectorAll('sky-help-inline').length,
+    ).toBe(0);
+
+    componentInstance.helpPopoverContent = 'popover content';
+    fixture.detectChanges();
+
+    expect(
+      fixture.nativeElement.querySelectorAll('sky-help-inline').length,
+    ).toBe(1);
+  });
```

Comment:
do we typically include popover title in this test?

Reply:
There isn't any logic handled in radio group's code change that would be affected by title so i left that out of this